eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/SynchronizedBiMap.java

```diff
@@ -291,6 +292,16 @@ public V forcePut(K key, V value)
         }
     }
 
+    @Override
+    public MutableBiMap<K, V> withMap(Map<? extends K, ? extends V> map)
+    {
+        synchronized (this.lock)
+        {
+            this.getDelegate().putAll(map);
```

Comment:
this could be this.putAll instead of this.getDelegate().putAll but I'm not sure if that matters